operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -248,43 +264,62 @@ impl<P: Pixel + GdalType> GdalDatasetWriter<P> {
 
         let tile_bounds = tile_info.spatial_partition();
 
-        let (upper_left, grid_array) = if self.output_bounds.contains(&tile_bounds) {
+        // compute the upper left pixel index in the output raster and extract the input data
+        let (output_ul_idx, grid_array) = if self.output_bounds.contains(&tile_bounds) {
```

Comment:
we could also do this using pixels right? We know the tile grid bounds from the TilingStrategy (+query rect) and each tile has a GridIdx of the tiling grid.

Reply:
I calculate the `contains` on the pixels now b0d17f67c2c79b8581d6ed3c9d6c4b2cd66fa10e